### ISSUE TITLE
Can't set scrollEnabled (on MapView) to true after setting it at false.

### ISSUE THREAD

**Issue by ethaqnix** [react-native-mapbox-gl/maps#211]
**Describe the bug**
Can't set scrollEnabled (on MapView) to true after setting it at false.

**To Reproduce**
```
<MapboxGL.MapView   
     scrollEnabled={this.state.scrollEnabled}
/>   
<Button   
     title={"scroll on / off"}
     onPress={() => { this.setState({ scrollEnabled: !this.state.scrollEnabled })} }
/>
```

**Expected behavior**
I expect to have the same behavior as other settings. being able to switch from disabled scroll to enabled.  

**Versions (please complete the following information):**
 - Platfrom: [e.g. Android, iOS]
 - SDK Version [e.g. 22]
 - React Native Version [e.g. 0.59]

**Additional context**
Other settings : 
- pitchEnabled
- rotateEnabled
- zoomEnabled 

works as expected

EDIT : sorry don't know what happen when filling the issue, I forget some information.

**Comment by mfazekas** [comment]
Fixed by #331

**Comment by scblason** [comment]
@mfazekas @ethaqnix Hi! We are experiencing this issue iOS in v8. It seems to be fixed for Android, but does not work for iOS. Do you know if there's a work around for that case?
